### ISSUE TITLE
PTSD

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#6820]
**Preferred term label:**
PTSD

**Synonyms**
post traumatic stress disorder

**Parent**
behavioral abnormality

Related to:
HP_0033676 HP_0033676

Need for long covid curation

**Comment by pnrobinson** [comment]
see https://hpo.jax.org/app/browse/term/HP:0033676
